### PR TITLE
Feature/redundancy state bridge feedback

### PR DESCRIPTION
**Pull request by jonnyarndt**
Revised readme.md, joinMap, controller, and simpl example to include calls for redundancy `role` vs `state`. Confirm content responses using Packet Sender. Remove ProcessJToken as it has been replaced.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,32 +79,68 @@ If `port` is only needed when overriding the default HTTP `80` or HTTPS `443`.
...
-| 2           | 1         | Power On (blackout)  | Digital | ToFromSIMPL  |
-| 50          | 1         | Is Online            | Digital | ToSIMPL      |
+| Join Number | Join Span | Description              | Type    | Capabilities |
+| ----------- | --------- | --------------------     | ------- | ------------ |
+| 1           | 1         | Power Off (blackout)     | Digital | ToFromSIMPL  |
+| 2           | 1         | Power On (blackout)      | Digital | ToFromSIMPL  |
+| 4           | 1         | Redundancy Role Main     | Digital | ToFromSIMPL  |
+| 5           | 1         | Redundancy Role Backup   | Digital | ToFromSIMPL  |
+| 6           | 1         | Redundancy Role Offline  | Digital | ToFromSIMPL  |
+| 7           | 1         | Redundancy State Main    | Digital | FromSIMPL    |
```
Comment (ndorin) [inline README.md:89]: These can't all be on join 7, right?

**Review by ndorin — APPROVED** [review]
Maybe fix up the readme to have accurate join number for the new joins.

**Comment by jonnyarndt** [comment]
@jtalborough or @ngenovese11 - Added for visibility and next steps.

The EPI workflow failed on the `Push_Nuget_Package (push)` YML, citing, `Error: Unable to resolve action `nuget/setup-nuget@v4`, unable to find version `v4`.

**Comment by jonnyarndt** [comment]
@jtalborough - I've re-ran the GitHub Actions a couple of times, both times failed. Any action on me?

**Comment by jtalborough** [comment]
> @jtalborough - I've re-ran the GitHub Actions a couple of times, both times failed. Any action on me?

It needed a push should be all set.  @jonnyarndt

**Pull request merged**